src/routers/workspaces.py

```
@@ -244,3 +249,77 @@ async def schedule_dream(
+                "application/json": {
+                    "schema": schemas.DialecticResponse.model_json_schema()
+                },
+                "text/event-stream": {},
+            },
+        },
+    },
+    dependencies=[Depends(require_auth(workspace_name="workspace_id"))],
+)
+async def chat(
+    workspace_id: str = Path(...),
+    options: schemas.WorkspaceChatOptions = Body(...),
```

Ruff (0.15.2)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)